app/src/screens/Terms.tsx

```diff
@@ -23,12 +24,15 @@ const digitalTrustHomeUrl = 'https://digital.gov.bc.ca/digital-trust/'
 const bcWebPrivacyUrl = 'https://www2.gov.bc.ca/gov/content/home/privacy'
 const digitalWalletPrivacyUrl = 'https://www2.gov.bc.ca/gov/content/governments/government-id/bc-wallet/privacy'
 
+export const TermsVersion = '1.0.1'
```

Comment:
please update to `2.0`

Reply:
got it, I'll do that once the bifold pr is merged bc I'll have to update the package.json anyway